appengine/pond/duck/online/js/online.js

```diff
@@ -169,6 +175,19 @@ Pond.Duck.Online.renderDuckInfo = function() {
       });
 };
 
+Pond.Duck.Online.computeRankRequest = function() {
+  if (this.status === 200) {
```

Comment:
I don't think we need this since BlocklyStorage.makeRequest already checks if the status === 200

Reply:
I modified BlocklyStorage to check whether the status is 2xx (success code). I send a 204 (not returning any content) when there is not any requests available for processing and 200 (ok) when there is.